Pipfile

```diff
@@ -8,6 +8,7 @@ pytest = "*"
 twine = "*"
 pyjwt = "*"
 Flask-SQLAlchemy = "*"
+Sphinx = "*"
```

Comment:
This is just for running the docs locally, this actually gets run automatically when the code is merged (we don't source control the build dir)